integrade/tests/api/v1/test_login.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+"""Test authentication with the server.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Cloud Meter
+:testtype: functional
+:upstream: yes
+"""
+from integrade import api
+from integrade.utils import uuid4
+
+import pytest
+
+
+@pytest.mark.skip
+def test_login():
+    """Test that we can login to the server.
+
+    :id: 2eb55229-4e1e-4d35-ac4a-4f2424d37cf6
+    :description: Test that we can login to the server
+    :steps: Send POST with username and password to the token endpoint
+    :expectedresults: Receive an authorization token that we can then use
+        to build our authentication headers and make authenticated requests.
+    :caseautomation: notautomated
+    """
+
+
+@pytest.mark.skip
+def test_logout(endpoint):
+    """Test that we can log out of the server.
+
+    :id: ca51b2a0-1e33-491d-8bb2-5e81d135424d
+    :description: Test that we can logout of the server and requests missing
+        a valid auth token are rejected.
+    :steps:
+        1) Log into the server
+        2) Logout of the server
+        3) Try an access the server
+    :expectedresults: Our request missing a valid auth token is rejected.
+    :caseautomation: notautomated
+    """
+
+
+def test_token():
+    """Given that we have a valid token, we can make requests.
```

Comment:
This is somewhat tested by the test_login which expects:

```
    :expectedresults: Receive an authorization token that we can then use
        to build our authentication headers and make authenticated requests.
```

How does this one differs from that?

Reply:
I see this as distinct because this starts with a valid token, where is "test_login" uses a username and password to attempt to get a token.